docs/radio.rst

```diff
@@ -0,0 +1,81 @@
+Radio
+*****
+
+.. py:module:: radio
+
+The ``radio`` module allows devices to work together via simple wireless
+networks.
+
+Functions
+=========
+
+.. py:function:: send(message)
+
+    Sends a message, expressed as a string or bytes. Sending a string is
```

Comment:
I don't like "expressed as a string or bytes", I would prefer "which can be a string or bytes object". The message (in the context of the API) is an object.